### PR TITLE
better naming and description of textindexer in behaviors control panel

### PR DESCRIPTION
**Pull request by jensens**
less clumpsy, easier to get,

### CODE REVIEW

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

File: plone/app/dexterity/textindexer/configure.zcml
Comment (mauritsvanrees) [inline plone/app/dexterity/textindexer/configure.zcml:11]: * `it's` -> `its`
* `got` -> `gets`

**Review by mauritsvanrees — APPROVED** [review]
LGTM except a comment about typos.
Build is green, can be merged when fixed (or fixed after merge).

File: plone/app/dexterity/textindexer/configure.zcml
Comment (jensens) [reply plone/app/dexterity/textindexer/configure.zcml:11]:
```suggestion
          description="Enables the enhanced full-text indexing for a content type ('plone.textindexer'). If a field is marked 'searchable', its content gets added to the 'SearchableText' index in the catalog."
```

**Pull request merged**
